ui/server.py

```diff
@@ -209,15 +209,15 @@ def getAppConfig():
 # TODO: change the vite config for public files
 @app.get('/ding.mp3')
 def read_ding():
-    return FileResponse(os.path.join(SD_UI_DIR, 'frontend/dist/ding.mp3'))
+    return FileResponse(os.path.join(SD_UI_DIR, 'frontend/assets/ding.mp3'))
```

Comment:
moved these to a sibling folder so they dont get "built" and added from the `build_src` to the `dist` folder